scripts/download_and_index.py

```
@@ -8,10 +8,10 @@
 sys.path.insert(0, str(script_dir))
 
 # Now import after path is set
-from mybgg.downloader import Downloader  # noqa: E402
-from mybgg.sqlite_indexer import SqliteIndexer  # noqa: E402
-from mybgg.github_integration import setup_github_integration  # noqa: E402
-from mybgg.config import parse_config_file, create_nested_config  # noqa: E402
+from gamecache.downloader import Downloader  # noqa: E402
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)